R/Driver.R

```diff
@@ -120,7 +120,7 @@ setMethod(
       addTaskCallback(function(expr, ...) {
         tryCatch({
           if (is.call(expr) &&
-              as.character(expr[[1]]) %in% c("<-", "=") &&
```

Comment:
I think an even better fix here would be to switch to rlang's `is_call()`: `is_call(expr[[1]], c("<-", "="))`. Looks like rlang is already an dependency of odbc.

Reply:
Did a couple tests, seems to work alright.